src/main/java/com/bynder/sdk/model/Media.java

```diff
@@ -105,6 +105,12 @@
      */
     private Map<String, String> thumbnails;
     /**
+     * focus point for the current media(image)
+     * map contains two key: 'x' and 'y' for the respective focus point coordinates.
+     */
+    private Map<String, Double> focusPoint;
```

Comment:
Sorry my bad, I forgot that when you request for a specific asset, you get the activeOriginalFocusPoint in the response, I thought this just happened for the media items.

Btw you need to add @SerializedName(value = "activeOriginalFocusPoint") on top of the class member and for the javadoc just say "Focus point of the media"

Reply:
just realized that, thanks

Reply:
need to test it briefly